src/services/organization/index.js

```diff
@@ -1,6 +1,6 @@
 const service = require('feathers-mongoose');
-const organization = require('./organization-model');
 const hooks = require('./hooks');
+const organization = require('./organization-model');
```

Comment:
The dependencies from npm will be put on top and the internal file dependencies come after. Then, we sort alphabetically in these groups.